File: C Programming/Factorial.c
// Program to find factorial of any number using C language
#include <stdio.h>
int main()
{
    int i, fact = 1;
    int n;
    printf("Enter the number : \n");
    scanf("%d", &n);
    for (i = 1; i <= n; i++)
    {
        fact *= i;
    }
    printf("The factorial of the number %d is %d", n, fact);
    return 0;
}

File: Java Programming/Java_Programming/src/Factorial.java
import java.util.Scanner;
public class Factorial {
    public static void main(String[] args){

        int  i, num;
        int fact = 1;
    Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number: ");
        num = sc.nextInt();

        for (i = 1; i <= num; i++){
           fact *= i;
        }

        System.out.println("The Factorial of " + num + " = " + fact);
    }
}
